runtime/compiler/optimizer/J9EstimateCodeSize.cpp

```diff
@@ -1145,8 +1142,11 @@ TR_J9EstimateCodeSize::realEstimateCodeSize(TR_CallTarget *calltarget, TR_CallSt
 
    bool wasPeekingSuccessfull = false;
 
-   if (nph.doPeeking() && recurseDown
-      || calltarget->_calleeMethod->convertToMethod()->isArchetypeSpecimen())
+   const static bool debugMHInlineWithOutPeeking = feGetEnv("TR_DebugMHInlineWithOutPeeking") ? true: false;
+   bool mhInlineWithPeeking =  !comp()->getOption(TR_DisableMHInlineWithoutPeeking); // TODO: turn off without peeking by default for now, will flip the option after sufficient testing
```

Comment:
can you clean-up the todo? I think this has been done...

Reply:
removed the TODO and turn on without peeking by default